src/plugins/dev_tools/public/index.scss

```diff
@@ -22,7 +22,13 @@
   flex-grow: 1;
 }
 
-.dataSourceSelector {
-  margin: 5px 10px 5px 5px;
+.devAppDataSourcePicker {
+  margin: 7px 8px 0 0;
   min-width: 400px;
 }
+
+.devAppTabs {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: space-between;
+}
```

Comment:
Can you explain more about why this wasn't achievable with `EuiFlexGroup` and `EuiFlexItem` as in the previous code? Because it likely means we need to open an issue in OUI to figure out how to address the need here.

Reply:
good catch, it's actually I forget to add the conditional render when I refactor this part of the code. I'll add it back, so if user disable datasource feature, datasource picker won't show, user will get the same old dev tool console experience.

Reply:
I don't think it's an issue with `EuiFlexGroup` or `EuiFlexItem`. Previously when we use `FlexGroup` to have 2 flex item, one is `EuiTabs`, one is `datasourcePicker`,  then there's this ugly line cut-off(comes with EuiTab).  We want the line to stretch all the way to be under the `DataSourcePicker`.

Reason for the cut-off is that we have 2 `flexItem`. But we want it to look better to avoid the cut-off, and the solution in this PR is by making `DataSourePicker` within EuiTab component, while removing the `FlexGroup` and `FlexItem`. But since `EuiTabs` itself is not inherently a flex component, I need to achieve that in CSS.

Reply:
I looked into this a bit - I think that the fundamental issue is that the current usage or `EuiTabs` here is not really the best component for the way we want the top of the page to work (`EuiPageHeader` with `tabs` and `rightSideItems` is more of what we want. But I think it makes the most sense to go forward with this hack for now, and then refactor it a part of a larger effort to convert the `dev_tools` and `console` plugins to be fully built with OUI components.